dhall-json/src/Dhall/JSON/Yaml.hs

```diff
@@ -64,22 +66,41 @@ parseQuoted =
             (   Options.Applicative.long "quoted"
             <>  Options.Applicative.help "Prevent from generating not quoted scalars"
             )
-                           
+
 {-| Convert a piece of Text carrying a Dhall inscription to an equivalent YAML ByteString
 -}
 dhallToYaml
-  :: Options
-  -> Maybe FilePath  -- ^ The source file path. If no path is given, imports
-                     -- are resolved relative to the current directory.
-  -> Text  -- ^ Input text.
-  -> IO ByteString
-dhallToYaml Options{..} mFilePath code = do
-  
-  let explaining = if explain then Dhall.detailed else id
+    :: Options
+    -> Maybe FilePath
+    -- ^ The source file path. If no path is given, imports
+    -- are resolved relative to the current directory.
+    -> Text
+    -- ^ Input text.
+    -> IO ByteString
+dhallToYaml mpath =
+    dhallToYamlWith Dhall.Context.empty Nothing mpath
+
+dhallToYamlWith
+    :: _
```

Comment:
Did you mean to leave these type holes here?

Reply:
No - I was waiting for `nix-build` to enlighten me via `stdout`, but apparently it never came